en/docs/deploy-and-publish/deploy-on-gateway/api-gateway/gateway-policies.md

```diff
@@ -0,0 +1,98 @@
+#Gateway Policies
+
+In WSO2 API Manager, managing Global Level Policies within the gateway infrastructure is made easier. This streamlines policy handling, eliminating the need for administrators to manually create XML-type policy files. This feature facilitates a user-friendly approach to handling policies, eliminates the requirement for manual XML creation and placement in specific directories, and enhances control by providing a clear interface for creating, deploying, and undeploying policies, offering administrators better control over policy management.
+
+!!! note
+    Policies added globally using the older approach: creating XML files following the naming pattern `WSO2AM--Ext--<DIRECTION>` and saving them in the `<API-M_HOME>/repository/deployment/server/synapse-configs/default/sequences` directory, will not be overridden by the global policies added through the UI. Initially, the system will execute the policies added via manual files and then proceed to execute the policies deployed from the UI.
+
+!!! warning
+    A specific gateway with active global policy deployments will apply to all the APIs deployed on that gateway. Similarly, this applies to tenants within the tenant available setup. Global policies will be applied to all APIs belonging to a specific gateway in the tenant corresponding to the logged-in tenant during the deployment of the global policy.
+
+!!!info
+    Although all managing actions can be taken by Admin users with specific scopes, API creators and publishers can view the meta information of global policies such as names, descriptions, policy mappings, and deployed gateway labels.
+
+## Step 1 - Create a gateway policy
+
+Follow the instructions provided below to create a gateway policy:
+
+1. {!includes/sign-in-publisher.md!}
```

Comment:
The content of the `sign-in-publisher.md` page does not seem to be right. Shall we manually add the steps here? No need to mention the test username and password that can be used.

Reply:
Fixed with https://github.com/wso2/docs-apim/pull/7526/commits/64bafa0aa418ed0dbbc05491694766f9d24f000b.
Thank you for your suggestion.